src/test/java/com/intel/gkl/testing_utils/TestingUtils.java

```diff
@@ -0,0 +1,24 @@
+package com.intel.gkl.testing_utils;
+
+import java.security.SecureRandom;
+import java.util.Arrays;
+
+public class TestingUtils {
+    public static final int MAX_SW_SEQUENCE_LENGTH = 32*1024-1; //2^15 - 1
+    public static final int MAX_SW_MATCH_VALUE = 64*1024; // 2^16
```

Comment:
Not used here

Reply:
I see you reference this in tests

Reply:
May be match this name to that used in SW java class MAX_SEQUENCE_LENGTH

Reply:
I added the `TestingUtils` class to hold all common functions and variables across all tests. #134 tests also use this class. I wanted to show that these values are specific to SmithWaterman only. I didn't want to make any confusion if someone would write new tests for PairHMM or Deflater/Inflater in the future.